CHANGELOG.md

```diff
@@ -1,3 +1,45 @@
+
+* An outstanding list of gesture issues and improvements were addressed in this release ([#9365](https://github.com/mapbox/mapbox-gl-js/pull/9365))
+    - You can now interact with multiple maps on the same page at the same page. For example, pinch rotate one map while touch panning a different one
+    - All gestures now have inertial. Previously touch rotation and mouse pitching did not have any
+    - It is now possible to switch fingers while panning the map without interrupting the pan. While panning: add a second finger, keep panning, remove the first, keep panning
+    - Touch rotation thresholding is improvement to make it harder to accidentally rotate when your fingers are very close together
+    - Fix map jump when releasing one finger after pinch zoom ([#9136](https://github.com/mapbox/mapbox-gl-js/issues/9136))
+    - Stop mousedown and touchstart from interrupting easeTo animations if interaction handlers are disabled ([#8725](https://github.com/mapbox/mapbox-gl-js/issues/8725))
+    - Stop mouse wheel from interrupting animations when `map.scrollZoom` is disabled ([#9230](https://github.com/mapbox/mapbox-gl-js/issues/9230))
+    - Fix `map.isZooming()` and `map.isMoving()` while zooming with Apple Magic mouse ([#6562](https://github.com/mapbox/mapbox-gl-js/issues/6562))
+  - _Changes to undocumented behavior_
+     - The previous gestures implementation prevented a camera change from actually happening if the interaction handler responsible for the change was disabled within the camera change event (`dragstart` or `rotatestart`). This behavior was not documented. It is no longer possible to prevent a camera change after it's event has been fired. It is possible some applications may be depending on this behavior. The correct way for these applications to selectively prevent camera changes is to listen to the `mousedown` or `touchstart` map event and call [.preventDefault()](https://docs.mapbox.com/mapbox-gl-js/api/#mapmouseevent#preventdefault) on it. For example, `map.on("mousedown", function(mapMouseEvent) { mapMouseEvent.preventDefault(); })`;
+     - The camera change events fired by the doubleClickZoom handler (`zoomstart`, `zoom`, `zoomend`, `movestart`, `move`, `moveend`) used to have undocumented properties such as `lngLat` and `point`. These have been removed
+* More precise collision detection for line labels ([#9219](https://github.com/mapbox/mapbox-gl-js/pull/9219))
+  - Improves placement under various camera orientation
+  - More tightly packed labels during placement, resulting in more visible labels
+  - Reduced memory footprint
+* Fix `GlyphManager` continually re-requesting missing glyphs ranges ([#8027](https://github.com/mapbox/mapbox-gl-js/issues/8027), fixed by [#9375](https://github.com/mapbox/mapbox-gl-js/pull/9375)) (h/t [oterral](https://github.com/oterral))
+* Remove Hershey Simplex font from our licence, which no longer needs attribution ([#9392](https://github.com/mapbox/mapbox-gl-js/issues/9392))
```

Comment:
I'm not sure I think that this needs to be our changelog. 

If we decide to include it, we should explain why it no longer needs attribution. I assume it's because we're no longer using it, in which case we could write: 

> Remove attribution for the Hershey Simplex font from license.txt, which this project no longer uses.

Reply:
Also, we should stick to the spelling of license that matches the name of the file

Reply:
I was not sure about this. Removed in 1f48c6396dbfc9caae467398fd73f8e779b2653f.